lib/config.js

```diff
@@ -340,6 +343,17 @@ config.start = function start() {
                          .pipe(errorLogger || logStream, {end: isPerWorker});
     worker.logFile = logStream;
   }
+
+  function setupExpresUsageForwarding(parentCtl) {
+    var debugExpress = require('./debug')('express-usage');
```

Comment:
We tend to use 1 debug per file, scoped on the file, and there is one already in config. If you use it, you can remove this function, and don't have to spread the code across two places.